tests/system_test.py

```diff
@@ -363,7 +365,7 @@ def wait_ports(self, **retry_kwargs):
 class Qdrouterd(Process):
     """Run a Qpid Dispatch Router Daemon"""
 
-    class Config(list, Config):
+    class Config(list, Config):  # type: ignore[misc]  # Cannot resolve name "Config" (possible cyclic definition)  # mypy#10958
```

Comment:
Known issue. We may rename the shadowed class eventually, or just leave this suppressed.